examples/Container/Frontend/Balancer/ConfigurableResolverFactory.cs

```diff
@@ -24,6 +24,8 @@ namespace Frontend.Balancer
 {
     public class ConfigurableResolverFactory : ResolverFactory
     {
+        private static readonly BalancerAttributesKey<string> HostOverrideKey = new BalancerAttributesKey<string>("HostOverride");
```

Comment:
I'm a little curious why we don't work directly with `"HostOverride"`, rather than wrapping and unwrapping it, but I'm assuming this is idiomatic.

Reply:
Its a pattern copied from the `HttpRequestOptions` collection. It's designed to make the collection feel a little more strongly typed.